deploy/serverpc/dlc/run_dlc.py

```diff
@@ -1,6 +1,7 @@
 import argparse
 from pathlib import Path
-from iblvideo import download_weights, dlc, __version__
+from iblvideo import download_weights
+from iblvideo.choiceworld import dlc
```

Comment:
we don't want to import `dlc` from `__init__.py`, otherwise `deeplabcut` will be imported which we don't want if we're in the litpose environment